frontend/src/themes/base.js

```diff
@@ -1,21 +1,59 @@
-import lightTheme from './lightTheme'
-import darkTheme from './darkTheme'
+import { common } from '@mui/material/colors'
 
-export function getThemeByName(theme) {
-  return themeMap[theme]
-}
+export const lightMode = 'light'
+export const darkMode = 'dark'
 
-export const themeMap = {
-  lightTheme,
-  darkTheme
-};
-
-export const themesList = [{
-  id: '1',
-  label: 'Light Theme',
-  value: 'lightTheme'
-}, {
-  id: '2',
-  label: 'Dark Theme',
-  value: 'darkTheme'
-}]
+export const getColorModeSettings = (mode = lightMode) => ({
+  palette: {
+    mode: mode,
+    primary: {
+      main: mode === 'light' ? '#262748' : common.white,
```

Comment:
I see you already have a constant for `light`

Reply:
done